### PR TITLE
Update basic.js

### PR DESCRIPTION
**Pull request by JustCursed**
the example doesn't work at the moment

### CODE REVIEW

File: app/snippets/docs/integration/libraries/nodejs/basic.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const Surreal = require('surrealdb.js');
+const Surreal = require('surrealdb.js').default;
```
Comment (kearfy) [inline app/snippets/docs/integration/libraries/nodejs/basic.js:1]: Thanks for this @JustCursed! I'd personally suggest to take a more ES6 like approach. Does that work for you?

```suggestion
const { default as Surreal } = require('surrealdb.js');
```

Comment (JustCursed) [reply app/snippets/docs/integration/libraries/nodejs/basic.js:1]: it doesn't work on common.js
`
const { default as Surreal } = require('surrealdb.js');
                ^^

SyntaxError: Unexpected identifier 'as'
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1166:20)
    at Module._compile (node:internal/modules/cjs/loader:1210:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1300:10)
    at Module.load (node:internal/modules/cjs/loader:1103:32)
    at Module._load (node:internal/modules/cjs/loader:942:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:83:12)
    at node:internal/main/run_main_module:23:47

Node.js v19.6.0
`
i can try to fix it in the library itself, but i might not be able to

Comment (kearfy) [reply app/snippets/docs/integration/libraries/nodejs/basic.js:1]: My bad! I slightly mixed up import statements and js object splitting there! I updated my example.
I'll see if I can get this stuff properly patched up in the JS library itself soon!

Comment (JustCursed) [reply app/snippets/docs/integration/libraries/nodejs/basic.js:1]: i can try write [surrealdb.node](https://github.com/surrealdb/surrealdb.node) with help napi-rs or ussing commonjs (without ts)

Comment (kearfy) [reply app/snippets/docs/integration/libraries/nodejs/basic.js:1]: I think we already have some plans on the table, but thanks for the offer!

**Pull request merged**